pkg/policies/opa/rego/aws/aws_apigatewayv2_api/apiGatewayMiconfiguredCors.rego

```diff
@@ -0,0 +1,8 @@
+package accurics
+
+apiGatewayMiconfiguredCors[api.id] {
+  api := input.aws_apigatewayv2_api[_]
+  cors := api.config.cors_configuration[_]
+  origins := cors.allow_origins[_]
+  not origins == ["*"]
```

Comment:
if "origins" does not exist, does it default to all domains?

Reply:
No. It defaults to only trust requests coming from the domain your API Gateway is serving traffic from.